pkg/python/pipenv/parse.go

```diff
@@ -10,11 +11,13 @@ import (
 )
 
 type lockFile struct {
-	Default map[string]dependency
-	Develop map[string]dependency
+	Default map[string]dependency `json:"default"`
+	//Develop map[string]dependency `json:"develop"`
```

Comment:
Do we want to remove it?

Reply:
We don't use developer dependencies, so i think we can remove it.

Reply:
Done